pathways/simulation.py

```diff
@@ -141,8 +140,10 @@ def simulation(
         if not shipment_actually_ok:
             if shipment_checked_ok:
                 missed_infestation_rate.append(shipment_infestation_rate(shipment))
+                total_missed_pests += Shipment.count_infested(shipment)
             else:
                 intercepted_infestation_rate.append(shipment_infestation_rate(shipment))
+                total_intercepted_pests += Shipment.count_infested(shipment)
```

Comment:
Although this works, it is unnecessarily complicated for Python. The `generate_shipment()` function in shipment generators returns a Shipment object, so the method/function is already available through that object:

```
total_intercepted_pests += shipment.count_infested()
```